samples/OpenQASM/OpenQasmHelloWorld.qasm

```diff
@@ -0,0 +1,28 @@
+/// # Summary
+/// OpenQASM Hello World sample
+///
+/// # Description
+/// This is one of the simplest Q# programs that contains quantum part.
```

Comment:
Q#?

Reply:
Now you know where I took that comment. :)